LICENSE.md

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+=====================
+
+Copyright (c) 2015 josephernest
```

Comment:
Thanks for this addition!

I'm rather new to pull requests. Can I modify a little something myself before merging? or do I need to show it here, and you do the modification?

    Copyright (c) 2015 Joseph Ernest